src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  sourceString.split(';').forEach((item) => {
+    const pairs = item.trim().split(':');
+    let key;
+    let value;
+
+    if (pairs.length === 2) {
+      key = pairs[0].trim();
+      value = pairs[1].trim();
+    }
+    obj[key] = value;
+  });
```

Comment:
The code should handle cases where there are empty declarations (e.g., consecutive semicolons without any content between them) to avoid adding empty keys to the object.